### DISCUSSION TITLE
Nice to have actual OSPO study cases of what they are doing day-to-day

### DISCUSSION THREAD

**Discussion by jerpelea** [todogroup/ospology#391] [Community Proposals]
A short OSPO presentation and pain points encountered by the presented OSPO followed by an open discussion with the other members regarding how they "fixed" the same issues.
I think that this kind of approach will encourage people to share their issues and problem solving experience

**Comment by anajsana** [comment]
## Context

TODO already has these use cases published https://todogroup.org/resources/case-studies/. But they are high level only. the main idea is to deep dive into day-to-day tasks and build new use case formats.

Examples:

* Share procedures
* Interactions with other teams (internal)
* Interactions with community (external)
* Share how they educate internally (e.g managers, engineers) and externally (to other OSPOs)

**Reply by anajsana** [reply]
Need also to decide which format is the best to leverage this (e.g live discussion via OSPOlogy or OSPOlogyLive Europe, written docs, etc)

**Comment by anajsana** [comment]
Speaking for myself, I believe adopting the idea @jerpelea outlines in this ticket could be beneficial as the core item in the 2024 OSPOlogy Europe Sync calls, once once the concept is fully defined and framed under the TODO mission.

**Comment by gkunz** [comment]
I support this proposal and I think it is very much in line with the mission of the TODO group.

I would also welcome to explicitly invite discussions about open and yet unsolved issues. I believe this would open up the floor to more presenters (i.e., including those in early OSPOs), and avoid the impression that only OSPOs which have solved "everything" are worth presenting. A great deal of value can be derived from talking about unsolved issues.

This may add constraints on how we run this to ensure compliance to the usual Chatham House rules.

**Comment by anajsana** [comment]
@gkunz @jerpelea The new OSPOlogy & TODO Touchpoint monthly Calls (Europe Chapter) have been scheduled for 2024, starting January 18th, from 3-4pm CET. As a starting point, the agenda includes a "work hour" section where the community could start sharing their day-to-day issues and exchange ideas and procedures.

https://todogroup.org/community/meetings/
